src/main/java/net/rptools/maptool/client/ui/campaignproperties/CampaignPropertiesDialog.java

```diff
@@ -738,7 +738,7 @@ private void commitSightMap(final String text) {
             gmOnly = false;
             owner = false;
           }
-          owner = gmOnly == true ? false : owner;
+          owner &= !gmOnly;
```

Comment:
Maybe 
owner = gmOnly ? false : owner;

owner &= !gmOnly;

makes me have to stop to think about it each time :)

Reply:
Intellij suggested the same thing; I went ahead and k-mapped it in my head.  I'll use the popular suggestion.

Reply:
Yeah. I had to stop and work through it in my head when I saw that.  This is not performance sensitive code so readability is probably more important.

Reply:
It wouldn't even be faster.  It's just a bad habit.